### ISSUE TITLE
Add logging to indicate the browser is unavailable 

### ISSUE THREAD

**Issue by StephenEsser** [cerner/terra-toolkit-boneyard#92]
# Feature Request

## Description

As a developer I want to log a meaningful message if the setViewportSize command fails. This message will help debug issues that occur in the before hook of the terra service.

We'll know we are done when a meaningful message is logged  if the setViewportSize command fails.

## Additional Context / Screenshots

Directly related to https://github.com/cerner/terra-toolkit-boneyard/issues/91.

A few teams are experiencing frequent BROWSER_TIMEOUT errors in the terra service before hook. These errors are related to the `browser.setViewportSize` command timing out. These errors were happening in Chrome.

Possible solution is to try catch the command in the [before service hook](https://github.com/cerner/terra-toolkit-boneyard/blob/3fdfbebfbeb68ef747c222539eadca3c18e4a5b2/src/wdio/services/TerraService.js#L73) or the [command itself](https://github.com/cerner/terra-toolkit-boneyard/blob/3fdfbebfbeb68ef747c222539eadca3c18e4a5b2/src/wdio/services/TerraCommands/viewport-helpers.js#L29).

Stack trace:

```
Error: A session is either terminated or not started

    at windowHandleSize([object Object]) - setViewportSize.js:83:26

FERROR: Session [e9dfca5689c81e4ebd82c86354115eaf] was terminated due to BROWSER_TIMEOUT

chrome

    at new RuntimeError (/app/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:143:12)

    at Request._callback (/app/node_modules/webdriverio/build/lib/utils/RequestHandler.js:318:39)

    at Request.self.callback (/app/node_modules/request/request.js:185:22)

    at Request.emit (events.js:198:13)

    at Request.EventEmitter.emit (domain.js:448:20)

    at Request.<anonymous> (/app/node_modules/request/request.js:1154:10)

    at Request.emit (events.js:198:13)

    at Request.EventEmitter.emit (domain.js:448:20)

    at IncomingMessage.<anonymous> (/app/node_modules/request/request.js:1076:12)

    at Object.onceWrapper (events.js:286:20)

    at IncomingMessage.emit (events.js:203:15)

    at IncomingMessage.EventEmitter.emit (domain.js:448:20)

    at endReadableNT (_stream_readable.js:1145:12)

    at process._tickCallback (internal/process/next_tick.js:63:19)

ERROR: Session [e9dfca5689c81e4ebd82c86354115eaf] was terminated due to BROWSER_TIMEOUT



    at new RuntimeError (/app/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:143:12)

    at Request._callback (/app/node_modules/webdriverio/build/lib/utils/RequestHandler.js:318:39)

    at Request.self.callback (/app/node_modules/request/request.js:185:22)

    at Request.emit (events.js:198:13)

    at Request.EventEmitter.emit (domain.js:448:20)

    at Request.<anonymous> (/app/node_modules/request/request.js:1154:10)

    at Request.emit (events.js:198:13)

    at Request.EventEmitter.emit (domain.js:448:20)

    at IncomingMessage.<anonymous> (/app/node_modules/request/request.js:1076:12)

    at Object.onceWrapper (events.js:286:20)

    at IncomingMessage.emit (events.js:203:15)

    at IncomingMessage.EventEmitter.emit (domain.js:448:20)

    at endReadableNT (_stream_readable.js:1145:12)

    at process._tickCallback (internal/process/next_tick.js:63:19)

Error: A session is either terminated or not started

    at windowHandleSize() - setViewportSize.js:72:17

Error: An unknown server-side error occurred while processing the command.

    at windowHandleSize() - setViewportSize.js:72:17
```
